### DISCUSSION TITLE
ADDING LOAD GRADUALLY

### DISCUSSION THREAD

**Discussion by stamatisg4** [e2nIEE/pandapower#2472] [Q&A]
### Feature Checklist

- [ ] Searched the [issues page](https://github.com/e2nIEE/pandapower/issues) for similar reports

- [ ] Read the relevant sections of the [documentation](https://pandapower.readthedocs.io/en/latest/about.html)

- [ ] Browse the [tutorials](https://github.com/e2nIEE/pandapower/tree/develop/tutorials) and [tests](https://github.com/e2nIEE/pandapower/tree/develop/pandapower/test) for usefull code snippets and examples of use

### Issue

Hello! 

I am new to using pandapower, and I would like your help with a problem I am facing. I have created a network and imported some time series for loads. What I want to do now is to gradually add loads to my buses, up to a limit that I will define. 

Thank you in advance!
Stamatis

### Label

- [ ] Relevant labels are selected

**Comment by pawellytaev** [comment]
Hey @stamatisg4 have you checked our hosting capacity tutorial? https://github.com/e2nIEE/pandapower/blob/develop/tutorials/hosting_capacity.ipynb

This might be a good starting point for a static hosting capacity analysis.

Then you can check out our time series tutorial and documentation https://github.com/e2nIEE/pandapower/blob/develop/tutorials/time_series.ipynb for relevant code snippets for time series analysis
